static/app/components/teamSelector.spec.jsx

```diff
@@ -135,5 +138,8 @@ describe('Team Selector', function () {
       expect.objectContaining({value: '2'}),
       expect.anything()
     );
+
+    // Wait for store to be updated from API response
+    await act(tick);
```

Comment:
nooo 😢

Reply:
is not possible to wait for the element to be rendered in the UI by using some of the `findBy` or `waitFor` queries?

Reply:
for all tests i've written so far i found a way to not use `tick`

Reply:
I dunno. I didn't want to spend a bunch of time trying to figure it out.

Reply:
Some of these are things like the render-prop style `Projects` util, that internally update state when the store changes, but do not actually render anything.

Unfortunately that still triggers the act warnings.